src/components/TypeLocationsPaper.js

```diff
@@ -184,8 +190,17 @@ const StyledResultPane = withTheme(withStyles(styles)(ResultPane));
 class TypeLocationsPaper extends Component {
   render() {
     const { classes, rights, title, onRefresh, onEdit, readOnly, location, ...others } = this.props;
```

Comment:
This line seems to be a little bit too long. Consider to restructure it to the same shape as the import in 14th line.